core/iwasm/aot/aot_runtime.c

```diff
@@ -1234,6 +1234,29 @@ aot_instantiate(AOTModule *module, bool is_sub_inst, WASMExecEnv *exec_env_main,
     }
 #endif
 
+#if WASM_ENABLE_GC != 0
+    if (!is_sub_inst) {
+        uint32 gc_heap_size = wasm_runtime_get_gc_heap_size_default();
+        AOTModuleInstanceExtra *extra =
+            (AOTModuleInstanceExtra *)module_inst->e;
+
+        if (gc_heap_size < GC_HEAP_SIZE_MIN)
+            gc_heap_size = GC_HEAP_SIZE_MIN;
+        if (gc_heap_size > GC_HEAP_SIZE_MAX)
+            gc_heap_size = GC_HEAP_SIZE_MAX;
+
+        extra->gc_heap_pool =
+            runtime_malloc(gc_heap_size, error_buf, error_buf_size);
+        if (!extra->gc_heap_pool)
+            goto fail;
+
+        extra->gc_heap_handle =
+            mem_allocator_create(extra->gc_heap_pool, gc_heap_size);
+        if (!module_inst->e->gc_heap_handle)
```

Comment:
How about using `extra->gc_heap_handle`? It is easier to understand.

Reply:
OK